.github/workflows/trivy-scan.yml

```diff
@@ -0,0 +1,88 @@
+name: Trivy CVE Dependency Scanner
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  scan-all:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v1
+        with:
+          go-version: "1.17"
+      - name: Build the kapp-controller artifacts
+        run: |
+          ./hack/install-deps.sh
+          ./hack/build.sh
+
+          # docker image
+          docker build -t docker.io/carvel/kapp-controller:${{ github.sha }} .
+      - name: Install Trivy
+        run: |
+          brew install aquasecurity/trivy/trivy
+
+          # download the sarif format template
+          git clone --depth 1 https://github.com/aquasecurity/trivy
```

Comment:
How is this used? Is there a different way of getting this than a git clone?

Reply:
So this is used by the following line:

`--format template --template "@trivy/contrib/sarif.tpl"`

The sarif.tpl is a template file that exists in the trivy repository, I couldn't find another way to get it. It's not built into the trivy scanner binary itself, so this seemed simplest

Reply:
Should we maybe check this file into the kapp-controller repo and update regularly? Not sure if we want to pull directly from main branch of trivy to get this.

Reply:
It felt cheaper to me for trivy folks to look after that template rather than us. Maybe we start with this, and if it ends up breaking a lot or they remove it, we can pull it into the repo? I am hesitant to take on the template ourselves, given a bunch of the carvel tools will be using the same template eventually. @danielhelfand

Reply:
Sounds good to me.